migrations/1737567411419_block-tx-total-size.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = pgm => {
+  pgm.addColumn('blocks', {
+    tx_total_size: {
+      type: 'int',
+      notNull: true,
+      default: 0,
+    },
+  });
+  pgm.sql(`
+    UPDATE blocks
+    SET tx_total_size = (
+      SELECT SUM(OCTET_LENGTH(raw_tx))
+      FROM txs
+      WHERE index_block_hash = blocks.index_block_hash
+    )  
+  `);
```

Comment:
My gut feeling is that this will result in a several hour long migration for most external deployments (e.g. exchanges). Seems like it would be fine to let the new column be nullable and we'd only have this available for new blocks.